dual_net.py

```diff
@@ -134,6 +134,40 @@ def should_log(i):
                     logger.add_summary(weight_summaries, global_step)
             self.save_weights()
 
+    def validate(self, tf_records, batch_size=128, logdir=None, init_from=None, num_steps=1000):
+        if logdir is None:
+            print("Need to run eval with a logdir")
+            return
+        if not logdir.endswith('_eval'):
```

Comment:
tensorboard's logdir can have subdirectories like train and eval, in which case they'll appear on tensorboard as separate curves named train and eval.

Reply:
yep yep, i had been shooting for something else but it was a bad idea.